Gordon360/Models/ViewModels/RecIM/ParticipantViewModel.cs

```diff
@@ -11,6 +11,8 @@ public class ParticipantViewModel
         public string Email { get; set; }
         public string? Role { get; set; }
         public string? Status { get; set; }
```

Comment:
Do you want to allow bulls for status? I don't think there's a particular way to do it, but in my mind, I would think having a default status would be better than allowing nulls.

Reply:
This is actually a good catch, it's not supposed to be null as it will never be null since the creation of Participant will automatically make an initial Status